src/main/frontend/pipeline-console-view/pipeline-console/main/hooks/use-steps-poller.spec.ts

```diff
@@ -49,7 +49,7 @@ afterEach(() => {
 });
 
 it("selects default step if URL param is missing", async () => {
-  const { result } = renderHook(() =>
+  const { result, unmount } = renderHook(() =>
```

Comment:
Required to fix the you need to wrap it in act warning, found on https://stackoverflow.com/a/73086378